docs/Readme.md

```diff
@@ -14,6 +14,10 @@ If you would like to see more examples of field data plug-ins, please visit our
 
 ## Change Log
 
+### AQTS 2020.3 - Framwork version 2.10
+- Added support for "OtherDischargeSection" channel measurements which can be used to record channel measures measured against any discharge monitoring method.
+- ManualGaugingDischargeSections no longer default to "Midsection" discharge method if no method is specified.
```

Comment:
I think it would be clearer if we call out the property that has been updated, something like:
`ManualGaugingDischargeSection.DischargeMethod is now optional and will no longer default to "MidSection" if omitted.`